docs/running-on-kubernetes.md

```diff
@@ -477,6 +477,22 @@ from the other deployment modes. See the [configuration page](configuration.html
   </td>
 </tr>
 <tr>
+  <td><code>spark.kubernetes.executor.labels</code></td>
+  <td>(none)</td>
+  <td>
+    Custom labels that will be added to the executor pods. This should be a comma-separated list of label key-value pairs,
+    where each label is in the format <code>key=value</code>.
```

Comment:
Ok that feels like a good path forward without completely compromising backcompat if we wanted.  Let's punt that to later.  https://github.com/apache-spark-on-k8s/spark/issues/322